python/rmm/tests/test_rmm.py

```diff
@@ -27,7 +27,7 @@
 from rmm.allocators.cupy import rmm_cupy_allocator
 from rmm.allocators.numba import RMMNumbaManager
 
-cuda.set_memory_manager(rmm.RMMNumbaManager)
+cuda.set_memory_manager(RMMNumbaManager)
```

Comment:
Appears the conflict was just [this change]( https://github.com/rapidsai/rmm/commit/1a75350e04649edd9b7a8186f5f600c7a66b4972#diff-edbfcbe7a4b6474859241cc360666a0d3962bffc01b6850c66ee0a75bc6850e0L36-R38 ). So including that here to resolve the conflict